web/src/apps/main.ts

```diff
@@ -87,118 +106,102 @@ export class App extends LitElement {
           ><img src="${logo}" alt="Stencila logo" width="28" height="28"
         /></a>
       </nav>
-      ${this.renderViewSelect()} ${this.renderThemeSelect()}
-      ${this.renderPrintLink()}
-    </header>`;
+    </header>`
   }
 
   private renderViewSelect() {
-    return html`
-      <label>
-        View
-        <select
-          @change=${(e: Event) =>
-            (this.view = (e.target as HTMLSelectElement).value as DocumentView)}
-        >
-          ${Object.entries(VIEWS).map(
-            ([view, desc]) =>
-              html`<option
-                value=${view}
-                title=${desc}
-                ?selected=${this.view === view}
-              >
-                ${view}
-              </option>`,
-          )}
-        </select>
-      </label>
-    `;
+    const clickEvent = (e: Event) => {
+      this.view = (e.target as HTMLButtonElement).dataset[
+        'value'
+      ] as DocumentView
+    }
```

Comment:
I like this approach of local functions for event handlers - keep things located together closely without cluttering the template.